app/views/assessments/show.html.erb

```diff
@@ -318,16 +318,15 @@
           <p>You cannot view previous submissions while an exam is in progress.</p>
       <% end %>
   <% end %>
-
+  <h4 class="section-title"><span class="white">
+    Submission Summary
+  </span></h4>
   <% if @submissions.size == 0 then %>
     No Submissions yet!
   <% else %>
-    <% for submission in @submissions.first(limit=3) do %>
-      <%= render partial: "submission_summary_row",
-        locals: { submission: submission,
-        download_access: download_access } %>
-    <% end %>
-    <%= link_to "See all submissions", {:action => "history"} %></li>
-
+    <%= render partial: "submission_history_table", locals: {download_access: download_access, limit: 3} %>
```

Comment:
Did GitHub mistook this line as a comment?

Reply:
I think so, it's actual code.